lib/serverengine/socket_manager_win.rb

```diff
@@ -15,12 +15,14 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 #
+require 'socket'
+require 'ipaddr'
```

Comment:
Hmm, I found that `socket_manager.rb` already requires `ipaddr` and `socket`.
This requirement is only for testing (or other purpose) which uses this file directly (without requirement of `socket_manager`).
So I think it doesn't make any problem not to require `ipaddr` in `_unix.rb`.